src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ReleaseDataFileServiceTests.cs

```diff
@@ -1839,14 +1842,14 @@ public async Task Upload()
                 privateBlobStorageService.Setup(mock =>
                     mock.UploadFile(PrivateReleaseFiles,
                         It.Is<string>(path =>
-                            path.Contains(FilesPath(releaseVersion.Id, FileType.Data))),
+                            path.Contains(FilesPath(releaseVersion.Id, FileType.Data, null))),
```

Comment:
Just questioning the change to `FilesPath`  in this PR which gives the folder path to a particular type of files, and is still used for this in the majority of cases but now has an additional optional param `filePath` which turns it into the path to a file when supplied.

It seems a bit strange that the method `FilesPath` (plural) is used to give the path to a file (singular).

I think rather than change this method you could have used the method which Mark already added `GovUk.Education.ExploreEducationStatistics.Content.Model.Extensions.FileExtensions.Path`

```csharp
        public static string Path(Guid rootPath, FileType type, Guid fileId)
        {
            return $"{FilesPath(rootPath, type)}{fileId}";
        }
```

and then these changes to supply the `null` param in all the existing cases where it's used for the folder path wouldn't be necessary. Appreciate that it's not an extension method even though it's in the extension namespace so that's something we could look into sorting out later on.

Reply:
Cheers, must have missed that method, I'll update.